canonical-websites/docs.snapcraft.io
# Replace discourse docs logic with module

The new module at
https://pypi.org/project/canonicalwebteam.discourse_docs/0.1/
is basically just a packaged-up version of the code in this
project.

So here I'm removing all the old code, and replacing it with this
module instead.

This is a significant step towards merging this app with the
snapcraft.io, so this documentation can ultimately be hosted
at https://snapcraft.io/docs.

QA
--

`./run`, browse around the site a lot. Nothing should have changed.

Check the tests pass. (The test coverage has unfortunately reduced, but I think this is hard to avoid)
